api/api_autogen/library/defaults/Cashloan_StandaloneBatteryCommercial.json

```diff
@@ -18,16 +18,16 @@
 		},
 		"SystemCosts": {
 			"add_om_num_types": 1,
-			"om_batt_capacity_cost": [15.000000],
+			"om_batt_capacity_cost": [12.000000],
```

Comment:
Similar question, checking that this is 12 when custom battery commercial and pv battery commercial are both 5.25

Reply:
Custom and PV both have some "standard" O&M captured in the main plant capacity numbers (e.g. rolling a truck to the site for inspections, monitoring software). Stand alone battery has a higher battery capacity number to capture these costs.